django_emarsys/util.py

```diff
@@ -0,0 +1,112 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+
+import logging
+import re
+
+from django.apps import apps
+from django.conf import settings
+
+log = logging.getLogger(__file__)
+
+
+def get_model(model_string):
+    chunks = model_string.rsplit('.', 1)
+    if len(chunks) == 2:
+        app_label, model_name = chunks
+    else:
+        app_label = None
+        model_name = model_string
+
+    return apps.get_model(app_label=app_label, model_name=model_name)
+
+
+def validate_settings():
+    validate_configuration({key: getattr(settings, key)
+                            for key in dir(settings)
+                            if hasattr(settings, key)})
```

Comment:
```
    @override_settings()
    def test_strange_dir_issue(self):
        delattr(settings, 'EMARSYS_ACCOUNT')
        self.assertTrue('EMARSYS_ACCOUNT' in dir(settings))
        self.assertFalse(hasattr(settings, 'EMARSYS_ACCOUNT'))
```
Only if the setting exists in the original settings and then is removed. But since `validate_settings()` might experience more testing, I'll leave the check in and describe it as you said.